### ISSUE TITLE
Timeline - Near Future

### ISSUE THREAD

**Issue by orgua** [nes-lab/shepherd#21]
Open Tasks 2022, see posts below for newer tasks

- [x] Programmer Mod #20 
- [x] Target PCB (msp430 + nRF) design + test
- [x] Target FWs DeepSleep
- postponed: Cape Mods & make operational, Web Frontend, backend & Api
- [x] Misc
  - [x] FW-node ID Patching
  - [x] PTP-Sync fix
  - [x] POE Problem with TP-Link Splitter

**Comment by orgua** [comment]
Tasks once deployment is done

- [ ] replace Fifo-system between userspace & pru
- [ ] allow static voltages for emulator without trace-recording
- [ ] gpio actuation in PRU
- [ ] update kernel to 5.15 
- [ ] time-sync improvement (issue #19)
- [ ] (web-) dataviewer for recorded traces

**Note**: these topics are pretty self contained and could be the base for student projects.
